cpp/daal/src/algorithms/dtrees/forest/classification/df_classification_predict_dense_default_batch_impl.i

```diff
@@ -969,23 +1035,23 @@ Status PredictClassificationTask<algorithmFPType, cpu>::predictAllPointsByAllTre
     ReadRows<algorithmFPType, cpu> xBD(const_cast<NumericTable *>(_data), 0, nRowsOfRes);
     DAAL_CHECK_BLOCK_STATUS(xBD);
     const algorithmFPType * const aX = xBD.get();
-    if (numberOfTrees > _MIN_TREES_FOR_THREADING)
+    if (numberOfTrees > _minTreesForThreading)
     {
         daal::static_tls<algorithmFPType *> tlsData([=]() { return service_scalable_calloc<algorithmFPType, cpu>(_nClasses * nRowsOfRes); });
 
         daal::static_threader_for(numberOfTrees, [&, nCols](const size_t iTree, size_t tid) {
             const size_t treeSize                = _aTree[iTree]->getNumberOfRows();
             const DecisionTreeNode * const aNode = (const DecisionTreeNode *)(*_aTree[iTree]).getArray();
-            parallelPredict(aX, aNode, treeSize, nBlocks, nCols, blockSize, residualSize, tlsData.local(tid), iTree);
+            parallelPredict(aX, aNode, treeSize, nBlocks, nCols, _blockSize, residualSize, tlsData.local(tid), iTree);
         });
 
-        const size_t nThreads  = tlsData.nthreads();
-        const size_t blockSize = 256;
-        const size_t nBlocks   = nRowsOfRes / blockSize + !!(nRowsOfRes % blockSize);
+        const size_t nThreads       = tlsData.nthreads();
+        const size_t localBlockSize = 256; // TODO: Why can't this be the class value _blockSize?
```

Comment:
This also looks like another one hyperparameter. As this value differs from the value of `_blockSize` and is used differently.
I think it can also be a target for a future extension of the hyperparameters set in RF.

Reply:
Yes, I would add it now, but I think in order to give it an appropriate name, we should first understand its exact role. So I'll leave it for future work.